Makefile

```diff
@@ -329,6 +329,7 @@ ifneq (,$(ACRNCMD))
     CONFIGS += $(CONFIG_ACRN)
 
     # acrn-specific options (all should be suffixed by "_ACRN")
+    DEFMAXVCPUS = 1
```

Comment:
this line will affect all hypervisors, add `_ACRN` suffix

Reply:
Thanks @devimc ! fixed it.